Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,21 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+
+namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            
             // Print each of the statistical output using Text Table with padding 3:
             // - FindHighestBalanceEver
```

Comment:
Comments are no longer needed- code speaks for itself.

Reply:
removed comments from main